performance/benchmarks/benchmark_spherical_representations.py

```diff
@@ -1,104 +1,109 @@
-"""
-@file  performance/benchmarks/benchmark_spherical_representations.py
-
-@author  Alexander Goscinski <alexander.goscinski@epfl.ch>
-
-@date   13 October 2019
-
-@brief benchmarks for spherical representations
-
-Copyright  2019 Alexander Goscinski, COSMO (EPFL), LAMMM (EPFL)
-
-rascal is free software; you can redistribute it and/or
-modify it under the terms of the GNU Lesser General Public License as
-published by the Free Software Foundation, either version 3, or (at
-your option) any later version.
-
-rascal is distributed in the hope that it will be useful, but
-WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
-Lesser General Public License for more details.
-
-You should have received a copy of the GNU Lesser General Public License
-along with this software; see the file LICENSE. If not, write to the
-Free Software Foundation, Inc., 59 Temple Place - Suite 330,
-Boston, MA 02111-1307, USA.
-"""
+# @file  performance/benchmarks/benchmark_spherical_representations.py
+#
+# @author  Alexander Goscinski <alexander.goscinski@epfl.ch>
+#
+# @date   13 October 2019
+#
+# @brief benchmarks for spherical representations
+#
+# Copyright  2019 Alexander Goscinski, COSMO (EPFL), LAMMM (EPFL)
+#
+# rascal is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation, either version 3, or (at
+# your option) any later version.
+#
+# rascal is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this software; see the file LICENSE. If not, write to the
+# Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+# Boston, MA 02111-1307, USA.
 
 # This file should be executed from the building directory
-# quick solution for loading 
-import sys, os
-sys.path.insert(0, os.getcwd())
```

Comment:
Hey you mentioned this today I think, but I did not get why this is not needed anymore?

Reply:
We set PYTHONPATH instead: https://github.com/cosmo-epfl/librascal/blob/e151317f6461fb440739d4a664818cf654c249ff/performance/benchmarks/CMakeLists.txt#L28-L30